rubble/src/att/mod.rs

```diff
@@ -535,7 +830,68 @@ impl<A: AttributeProvider> AttributeServer<A> {
             }
         }
 
+        // Should be in its own module?
```

Comment:
Yeah it might be time to split this up. All the data type (de)serialization should be in its own module, and the actual server impl should probably also get its own module, so that `att` mainly just contains submodules that do the interesting bits.